azure-pipelines.yml

```diff
@@ -6,13 +6,19 @@
 #
 
 variables:
+    # clean the local repo on the build agents
+  - name: Build.Repository.Clean
+    value: true
+  - name: _PublishUsingPipelines
+    value: true
+  - name: _DotNetArtifactsCategory
+    value: WINDOWSDESKTOP
+  - name: _DotNetValidationArtifactsCategory
```

Comment:
I guess this will be removed before merging?

Reply:
I'm assuming the question is about _DotNetValidationArtifactsCategory. _DotNetArtifactsCategory=WINDOWSDESKTOP is required. 

Does it have to be removed? I was going to leave it in. I would imagine it comes into play only when channel=validation.

Reply:
Yes I was refering to `_DotNetValidationArtifactsCategory` -- it doesn't hurt leaving it in, just to make it cleaner and not have unused variables, however on a second thought, this might come into play when testing new validations or new features for publishing from arcade that need to be adopted.